utilities/common.py

```diff
@@ -117,12 +117,15 @@
 
 def get_binning_fakes_pt(min_pt, max_pt):
     edges = np.arange(min_pt,32,1)
-    edges = np.append(edges, [e for e in [33,36,40,46,56] if e<max_pt][:-1])
+    edges = np.append(edges, [e for e in [35,38,41,44,47,50,53,56] if e<max_pt][:-1])
```

Comment:
np.arange goes to 31 so the bins are [...,30,31,35,38,...] so there is one bin with width 4, maybe add 33 ? so that there are two bins with width 2GeV In between?